lib/id3c/db/session.py

```diff
@@ -85,6 +85,11 @@ def rollback(self):
         """Proxy for the underlying connection's ``rollback`` method."""
         return self.connection.rollback
 
+    @property
+    def close(self):
+        """Proxy for the underlying connection's ``close`` method."""
+        return self.connection.close
```

Comment:
Does `close` work even if the connection is already closed? If not, there should be a check.

Reply:
Yes, you can call `close` as many times as you want. If you try to use the connection after closing it, however, an error will be thrown.

This is the behaviour of the underlying psycopg2 library, which ID3C is just proxying here.